i18n/locales/en.ts

```diff
@@ -2737,49 +2737,52 @@ Stack Hint: To spend before the secret is revealed, Vanderpoole uses his signatu
     },
     opening_a_channel_four: {
       title: 'Multisig',
       nav_title: 'Multisig',
       heading_one: 'Multisig',
-      paragraph_one: `This game will have to start with a 2-of-2 multisig, confirmed on the blockchain. Once we have that we can figure out a way to make that single on-chain transaction work harder. To do more with less.`,
+      paragraph_one: `Before paying Laszlo, let's first get the funds into a 2-of-2 multisig, confirmed on the blockchain. Then we can figure out a way to make that single on-chain transaction work harder—to do more with less.`,
+      heading_two: 'Instructions',
       multisig_one: 'Fill in the output amount and the output script',
       multisig_two: 'Sign the input by clicking "Sign"',
-      heading_two: 'Hints',
+      heading_three: 'Hints',
       hint_one_a: `You and Laszlo want to work together in the future to spend output 0 with <br/> `,
       hint_one_b: `<span className="rounded-sm px-1.5 py-1 h-[28px] font-mono bg-[#0000004D] m-0.5 text-base"> 0 SIG(LASZLO) SIG(YOU) </span>`,
+      hint_two: `Don't forget to set aside miner fees`,
     },
     opening_a_channel_five: {
-      title: 'Refund Protection in Multisig',
-      nav_title: 'Refund Protection in Multisig',
-      paragraph_one: `—LASZLO:  "OK, thanks, this is a good start. But what if I walk away from the table and you never see me again? I could broadcast this transaction, and your 100,000 satoshis would be stuck in a 2-key multisig from which you could never recover.`,
+      title: 'Refund protection in multisig',
+      nav_title: 'Refund protection in multisig',
+      paragraph_one: `—LASZLO:  "OK, thanks. This is a good start. But what if I walk away from the table and you never see me again? I could broadcast this transaction, and your 100,000 satoshis would be stuck in a 2-key multisig from which you could never recover.`,
       paragraph_two: `Do yourself a favor; before signing this, make a refund transaction to know you can recover your money."`,
     },
     updating_the_state_one: {
       title: 'The refund',
       nav_title: 'The refund',
       heading_one: 'The refund',
-      paragraph_one: `A new tab appears: <span className="rounded-sm px-1.5 py-1 h-[28px] font-mono bg-[#0000004D] m-0.5 text-base">Refund</span> which starts another TX template on the same screen, with an arrow from the <span className="rounded-sm px-1.5 py-1 h-[28px] font-mono bg-[#0000004D] m-0.5 text-base">Multi-Sig</span> output to this TX input.`,
+      paragraph_one: `A new tab appears: <span className="rounded-sm px-1.5 py-1 h-[28px] font-mono bg-[#0000004D] m-0.5 text-base">Refund</span> which starts another TX template on the same screen, with an arrow from the <span className="rounded-sm px-1.5 py-1 h-[28px] font-mono bg-[#0000004D] m-0.5 text-base">Multisig</span> output to this TX input.`,
```

Comment:
i noticed this kind of copy later on in the chapter -- its my fault for writing the challenge this way, its a mix of actual copy for the user and "stage directions" for design. But if we act like the right side of the screen actually IS the user's ePhone Infinity, then it makes sense

Reply:
Oooo! I like the idea of mentioning the new tab is on your ePhone Infinity. I updated it as part of https://github.com/saving-satoshi/saving-satoshi/pull/1174/commits/5dbeca111b179538b786d1804a51e4080a6f9aae